docs/manage/CLI.md

```diff
@@ -190,6 +193,107 @@ The `logout` command ends an authenticated CLI session
 viam logout
 ```
 
+### module
+
+The `module` command allows to you to manage custom modules.
+This includes:
+
+* Creating a new custom modular resource on your local filesystem
+* Updating an existing module with new changes
+* Uploading a new module to the Viam Registry
+* Updating an existing module in the Viam Registry
+
+```sh {class="command-line" data-prompt="$"}
+viam module create --organization=<org name> --name <module-id>
```

Comment:
the syntax of these commands are not correct. Please see this doc for the instructions we are using today: https://docs.google.com/document/d/1HSOV6ncMv5QkpdWOqp-f_RnUxHz4QnrnchjLLjN69MQ/edit


users can put in either an org-id OR public-namespace for each step